java/tests/generated/com/facebook/yoga/YGAbsolutePositionTest.java

```diff
@@ -70,6 +70,135 @@ public void test_absolute_layout_width_height_start_top() {
     assertEquals(10f, root_child0.getLayoutHeight(), 0.0f);
   }
 
+  @Test
+  public void test_absolute_layout_width_height_left_auto_right() {
+    YogaConfig config = YogaConfigFactory.create();
+
+    final YogaNode root = createNode(config);
+    root.setPositionType(YogaPositionType.ABSOLUTE);
+    root.setWidth(100f);
+    root.setHeight(100f);
+
+    final YogaNode root_child0 = createNode(config);
+    root_child0.setPositionType(YogaPositionType.ABSOLUTE);
+    root_child0.setPositionAuto(YogaEdge.LEFT, YogaConstants.AUTOf);
```

Comment:
This doesn't seem valid

Reply:
More generally, I think these tests assume JNI bindings for the Java Node exist to the new API.

Annoyingly, we don't run Java build or tests in GitHub Actions right now, but it is run when we import it into Phabricator, so it's usually pretty easy to pattern match an existing binding.

Reply:
Okay, I can see that there is no `setPositionAuto` function for Java too.

Reply:
Yep! That and generating `YogaConstants.AUTOf` (with the extraneous "f")